WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -3028,7 +3088,7 @@ static GSourceFuncs _handlerIntervention =
 
             // Allow mixed content.
             bool enableWebSecurity = _config.Secure.Value();
-#if WEBKIT_CHECK_VERSION(2, 38, 0)
+#if WEBKIT_CHECK_VERSION(2, 40, 0)
```

Comment:
I thought `disable-web-security` arrived with version 2.40, like it says here: https://github.com/WebPlatformForEmbedded/WPEWebKit/blob/8a6a01511e58f0a42c98084938e759938c4ec1f8/Source/WebKit/UIProcess/API/glib/WebKitSettings.cpp#L1663
But if we're sure it can be used under in 2.38 and this is just a typo in this comment, then I'll revert it to 2.38.

Reply:
Yes, it needs to be 2.38.